app/src/main/java/org/mozilla/tv/firefox/channels/BundleTilesStore.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.channels
+
+import android.app.Application
+import org.json.JSONArray
+
+private const val BUNDLED_PINNED_TILES_DIR = "bundled"
+private const val PINNED_TILES_JSON_PATH = "$BUNDLED_PINNED_TILES_DIR/bundled_tiles.json"
+
+enum class BundleType {
+    PINNED_TILES
+}
+
+/**
+ * [BundleTilesStore] is responsible for fetching bundled tiles data from /assets/ with
+ * [BundleType]
+ */
+class BundleTilesStore(private val applicationContext: Application) {
+    fun getBundledTiles(type: BundleType): JSONArray {
```

Comment:
While we're moving things around anyway, I feel like we should be doing type conversions inside this class.  We lose a lot of type safety and clarity by exposing a `JSONArray`, I'd rather do that conversion on the edge between the app and storage (that is, here).

TL;DR: I think this should be a private function, then we can expose a `getPinnedTiles` method that calls this, converts, and returns a `List<ChannelTile>` (or `List<PinnedTile>` if that's unworkable).

Reply:
Yes I agree with this, I'll give some think on it